### PR TITLE
ex/websites/signupz

### PR DESCRIPTION
**Pull request by skyl**
## What

Basic signup site for now. Someday with private signups using Zcash

## Why

I need a signup site anyways. But, it doesn't seem like the existing products are mindful of privacy. So, I'm going to be stubborn and build my own signup site for emails. Someday this site could be even more private with Zcash signups.

### CODE REVIEW

File: ex/websites/signupz/templates/base.html [hunk 1]
```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
```
Comment (skyl) [inline ex/websites/signupz/templates/base.html:1]: delete, rename index

**Review by birdify — COMMENTED** [review]
Artwork looks dope, where'd you find that?  The UI for this site looks sweet!

**Comment by skyl** [comment]
> Artwork looks dope, where'd you find that? The UI for this site looks sweet!

yeah, I just googled it. It was on some "freepics" type site. But, should figure out who the artist is and do some combination of a) credit b) pay c) teamup

File: ex/websites/signupz/README.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Signups
+
+Example website that can collect pre-launch signup information.
+
+Currently email-only.
+But, could incorporate Zcash and other payments as condition of signup.
```
Comment (skyl) [inline ex/websites/signupz/README.md:6]: (or as a private way to sign up for communications privately and anonymously)

File: ex/websites/signupz/requirements.txt [hunk 3]
```diff
@@ -0,0 +1,2 @@
+Django==3.2
```
Comment (skyl) [inline ex/websites/signupz/requirements.txt:1]: Tested with `Python 3.9.2`

**Pull request merged**